addons/test_mail_full/tests/test_rating.py

```diff
@@ -20,88 +20,104 @@ def setUpClass(cls):
             'name': 'Test Rating',
             'user_id': cls.user_admin.id,
         })
+        cls.record_rating_thread = cls.env['mail.test.rating.thread'].create({
+            'partner_id': cls.partner_1.id,
+            'name': 'Test rating without rating mixin',
+            'user_id': cls.user_admin.id,
+        })
 
 
 @tagged('rating')
 class TestRatingFlow(TestRatingCommon):
 
     def test_initial_values(self):
-        record_rating = self.record_rating.with_env(self.env)
-        self.assertFalse(record_rating.rating_ids)
-        self.assertEqual(record_rating.message_partner_ids, self.partner_admin)
-        self.assertEqual(len(record_rating.message_ids), 1)
+        for record_rating in [self.record_rating, self.record_rating_thread]:
+            record_rating = record_rating.with_env(self.env)
+            self.assertFalse(record_rating.rating_ids)
+            self.assertEqual(record_rating.message_partner_ids, self.partner_admin)
+            self.assertEqual(len(record_rating.message_ids), 1)
 
     @users('employee')
     @mute_logger('odoo.addons.mail.models.mail_mail')
     def test_rating_prepare(self):
-        record_rating = self.record_rating.with_env(self.env)
-
-        # prepare rating token
-        access_token = record_rating._rating_get_access_token()
-
-        # check rating creation
-        rating = record_rating.rating_ids
-        self.assertEqual(rating.access_token, access_token)
-        self.assertFalse(rating.consumed)
-        self.assertFalse(rating.is_internal)
-        self.assertEqual(rating.partner_id, self.partner_1)
-        self.assertEqual(rating.rated_partner_id, self.user_admin.partner_id)
-        self.assertFalse(rating.rating)
+        for record_rating, desc in [(self.record_rating, 'With rating mixin'),
```

Comment:
picky std: when possible use tuple and not list (same below)

Reply:
yes, I suppose it is better for performance